runtime/doc/options.txt

```diff
@@ -8059,6 +8059,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	If a window has 'winfixbuf' enabled, 'switchbuf' is currently not
 	applied to the split window.
 
+						*'switchtab'* *'swt'*
+'switchtab' 'swt'	string	(default "")
+			global
+	This option controls the behavior when switching tabs, for example when
```

Comment:
Does this really handle *switching tabs*? Isn't it rather about *closing tabs* instead?